lib/sisimai/bite/email/domino.rb

```diff
@@ -0,0 +1,174 @@
+module Sisimai::Bite::Email
+  module Domino
+    # Sisimai::Bite::Email::Domino parses a bounce email which created by IBM
+    # Domino Server. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Domino.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :subject => %r/\ADELIVERY FAILURE:/,
+      }
+      Re1 = {
+        :begin   => %r/\AYour message/,
+        :rfc822  => %r|\AContent-Type: message/delivery-status\z|,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->